CsvParser/csvParser.go

```diff
@@ -40,14 +41,22 @@ func CsvParser(filePath string) [100]incident.DefaultIncident {
 	// 	log.Fatal(err)
 	// }
 
-	for i := 0; i < 100; i++ {
+	if len(numIncidents) > 1 {
+		log.Fatal("too many arguments!")
+	}
 
+	var i = 0
+	for {
 		rec, err = reader.Read()
+		if len(numIncidents) == 1 && i == numIncidents[0] {
```

Comment:
You don't really need to check if len(numIncidents) == 1 since you have done that on line 44

Reply:
No....I only checked if the number of specifications inputted is greater than one (in which case it's an error...since we can't generate 4,3 incidents. we need exactly one input). I need to check if the user inputted one integer, because if they did not input anything I generate all incidents. I need to check if the user inputted one integer to stop producing incidents once